gapic/src/main/com/google/gapid/models/Memory.java

```diff
@@ -685,10 +685,9 @@ private static StructNode removeExtraLayers(StructNode root) {
           && root.children.get(0).hasChildren()) {
         root = root.children.get(0);
       }
-      // Remove the inner layer for PSEUDONYM type. E.g. {[()]} -> {()}.
-      if (tyCase == TyCase.PSEUDONYM && root.hasChildren() && root.children.size() == 1
-          && root.children.get(0).hasChildren()) {
-        root.children = root.children.get(0).children;
+      // Treat PSEUDONYM types as leaves, and remove any children they may have had.
+      if (tyCase == TyCase.PSEUDONYM) {
+	root.children.clear();
```

Comment:
Sounds good to treat them as leaves! Just a small nit for indenting here.

Reply:
Misconfigured editor, I usually don't touch the Java... thanks